packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -75,10 +75,6 @@ const GlobalHeader = ({ className }) => {
 
   const hideLogoText = useMedia({ maxWidth: '655px' });
   const useCondensedHeader = useMedia({ maxWidth: '585px' });
-  const inDocsSite = [
-    'https://docs.newrelic.com',
-    'https://docs-preview.newrelic.com',
-  ].includes(siteUrl);
```

Comment:
If we're no longer relying on `siteUrl`, then we can probably remove it from the query above (line 59 and 73).

Reply:
will do